### PR TITLE
Update responsive header (and standardize styling of hero and page spacing)

### PR DESCRIPTION
**Pull request by marteki**
This updates the site to use one of the standard navigation headers for consumerfinance.gov. It also updates the hero and spacing on the page to use Capital Framework standards. EDIT: It now also adds a link to switch between languages.

## Additions

- Add link to switch between languages

## Removals

- Removed redundant references to nemo and nemo_shim in HTML, LESS, and JS files

## Changes

- Changes to the Django base templates, courtesy of @higs4281 
- Changes to claiming.html and claiming.less

## Testing

- Pull down the `newbase` branch. Load up the site, and give it a whirl.
  - The header should look correct, and work as expected at all screen sizes.
  - The hero should display properly.
  - At under 1170px screen width, the page should have margins in line with the hero's internal margins.

## Review

- @niqjohnson @mistergone @higs4281

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by higs4281** [comment]
:100: great job. tested in UnityBox

File: retirement_api/templates/claiming.html [hunk 1]
```diff
@@ -1,459 +1,418 @@
...
-    htmlTag.className = classes.replace("no-js", "");
-  }
-</script>
-    <div id="page">
-
+{% load static from staticfiles %}
+{% load i18n %}
+{% block title %} {% trans page.title %} {% endblock %}
+{% if es %}
+  {% block languge %}es{% endblock %}
```
Comment (higs4281) [inline retirement_api/templates/claiming.html]: This block is no longer needed, and it's typo-ed to boot. I'll remove.

File: retirement_api/templates/claiming.html [hunk 2]
```diff
@@ -560,28 +515,8 @@ <h3 class="lifestyle-question">{% trans questions.longlife.question %}</h3>
             </nav>
           </section>
         </div>
```
Comment (niqjohnson) [inline retirement_api/templates/claiming.html]: I think we can remove this whole `div.nemo`. Looks like it just contains an extra copy of the subnav now that the subnav has been moved to the base template.

**Comment by niqjohnson** [comment]
This is looking great in every browser I tried! One line note above about getting rid of a duplicate subnav.

Also, running the UnityBox version through an HTML validator give an unclosed tag warning (couldn't track down where it was at a glance, though):

errors

And two notes that don't need to hold up this PR:

1. This is going to cause some merge conflicts with #148 and #150. This update is more important, so I'll hold off on merging those until this one is in.

2. The "about" template will need to get reworked a little to use the common base (it's unstyled in this branch).

**Comment by marteki** [comment]
Got the duplicate subnav, and found the missing closing tag! Thank you!!!

Adding in the Spanish toggle to this PR as well, as soon as I align it properly on small and medium screens. Stand by...

**Comment by marteki** [comment]
Please review again? I added the toggle for switching between English and Spanish!

**Comment by Scotchester** [comment]
Just to make sure I understand correctly, if you're _not_ running the standalone version of the app, it extends `base_update.html` from django-cfgov-common, right?

**Comment by higs4281** [comment]
@Scotchester you got it. 

with this PR, our pages extend common's `front/base_update.html` unless we're running standalone.

**Comment by Scotchester** [comment]
Rock on! :metal: 

[N.B.: That is not an indicator that I have conducted any code review, nor plan to.]

**Comment by niqjohnson** [comment]
The subnav and HTML validation fixes look good to me.

The Spanish toggle is working for me on both the English and Spanish pages. I did notice a slight discrepancy (color and underline style) between the comp for the toggle and the working code:

Comp | Code
-------- | -------
comp | code

Also, the working code uses the usual light blue hover style, which is hard to read on top of the orange hero background.

Also also, this is just me, but the toggle feels a little floaty at screen sizes where the hero image is hidden:

medium

I'll defer to @caheberer on all of those issues, though :smile:

**Comment by marteki** [comment]
Ack, I forgot about the blue being not accessible on orange. I'll fix that.

- Marteki

**Comment by marteki** [comment]
Changed the toggle link color and underline styling. (@niqjohnson &ndash; I continued to use `border-style` instead of changing it to the underline-aligned-with-the-baseline that's shown in the comp, so the code would tie more closely to our link standards.)

**Comment by marteki** [comment]
@caheberer is the one who asked that the link hang out on the right on medium screen widths. :stuck_out_tongue: 

BUT! I just realized from your screenshot that the hero text isn't going full-width anymore on the medium screens. Another fix incoming.

**Comment by marteki** [comment]
Confirmation from team members that the link styling is fixed, and this is ready to go!

**Pull request merged**